### PR TITLE
added in selectors for automated login tests and settings tests

### PR DESCRIPTION
**Pull request by noahgildersleeve**
removed extra ID that wasn't used

### CODE REVIEW

**Comment by noahgildersleeve** [comment]
So I see some commits in there that looked a little odd. I think it was based on trying to rebase from the upstream master branch to get everything in my fork. I'm not sure why it was showing up like that though.

**Comment by n313893254** [comment]
As every rebase is a force-push to the master branch. we need to cherry-pick the commits based on the new master branch.
Here is an example to rebase does not need to close the PR:

```bash
git branch -D master
git fetch origin
git checkout origin/master
git checkout -b master
git cherry-pick 4adbefd
git branch -D pom-selector-updates
git checkout -b pom-selector-updates
git push fork -f
```

**Addictional Context**
[.git/config]
```bash
[remote "origin"]
        url = git@github.com:harvester/dashboard.git
        fetch = +refs/heads/*:refs/remotes/origin/*
[remote "fork"]
        url = git@github.com:noahgildersleeve/dashboard.git
        fetch = +refs/heads/*:refs/remotes/fork/*
```

**Comment by noahgildersleeve** [comment]
I fixed it. Thanks for the help.

**Review by WuJun2016 — CHANGES_REQUESTED** [review]
@noahgildersleeve  Spelling error not fixed.

File: pages/auth/setup.vue [hunk 1]
```diff
@@ -334,18 +334,18 @@ export default {
...
+              <Checkbox id="checkbox-eula" v-model="eula">
                 <template #label>
                   <t k="setup.eula" :raw="true" :name="productName" />
                 </template>
               </Checkbox>
             </div>
           </template>
 
-          <div class="text-center mt-20">
+          <div id="submit" lass="text-center mt-20">
```
[the remark this answers is not in the document]
Comment (noahgildersleeve) [reply pages/auth/setup.vue]: I updated it. Thanks.

**Comment by noahgildersleeve** [comment]
Fixed.

**Review by WuJun2016 — APPROVED** [review]
LGTM

**Pull request merged**